monkey_island/cc/island_logger_default_config.json

```diff
@@ -0,0 +1,33 @@
+{
+    "version": 1,
+    "disable_existing_loggers": false,
+    "formatters": {
+        "simple": {
+            "format": "%(asctime)s - %(filename)s:%(lineno)s - %(funcName)10s() - %(levelname)s - %(message)s"
+        }
+    },
+
+    "handlers": {
+        "console": {
+            "class": "logging.StreamHandler",
+            "level": "DEBUG",
+            "formatter": "simple",
+            "stream": "ext://sys.stdout"
```

Comment:
Why not stderr?

Reply:
its just common practice

Reply:
It can be confusing to output both debug and errors alongside INFO level to stderr, thats why the split is being made on a higher level design. also splitting the streams is a bit of an overkill.